addons/mail/static/tests/models/qunit_test.js

```diff
@@ -11,13 +11,6 @@ registerModel({
             inverse: 'qunitTestOwner',
             isCausal: true,
         }),
-        composer: one('Composer', {
-            isCausal: true,
-        }),
-        composerView: one('ComposerView', {
-            inverse: 'qunitTest',
-            isCausal: true,
-        }),
         followerListMenuView: one('FollowerListMenuView', {
```

Comment:
followerListMenuView, and maybe notificationListView and others are also no longer used since previous PR

Reply:
`notificationListView` will come in a later PR :+1: